jaxrs/src/main/java/io/airlift/jaxrs/tracing/TracingServletFilter.java

```diff
@@ -73,6 +74,13 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                 .setAttribute(ClientAttributes.CLIENT_ADDRESS, request.getRemoteAddr())
                 .setAttribute(NetworkAttributes.NETWORK_PROTOCOL_NAME, "http");
 
+        // https://github.com/jetty/jetty.project/blob/jetty-12.0.x/jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletContextRequest.java#L63
+        String sessionId = (String) request.getAttribute("jakarta.servlet.request.ssl_session_id");
```

Comment:
Is this guaranteed, or an implementation detail of Jetty?

Reply:
It works in Tomcat, Jetty, Undertow.

Reply:
When moved in the last commit, I can now reference the public static const from Jetty directly.